x-pack/plugins/observability/server/routes/slo/route.ts

```diff
@@ -14,8 +22,36 @@ const createSLORoute = createObservabilityServerRoute({
     tags: [],
   },
   params: createSLOParamsSchema,
-  handler: async ({ context, request, params }) => {
-    return { success: true };
+  handler: async ({ context, request, params, logger, spacesService }) => {
```

Comment:
Temporary handler. I'd like to extract that and provide only a CreateSLO service to call from here. Need to investigate a bit how and when some of the dependencies are available and can be injected.